src/components/CreateVmWizard/steps/BasicSettings.js

```diff
@@ -357,6 +366,7 @@ class BasicSettings extends React.Component {
               items={isoList}
               selected={data.isoImage || '_'}
               onChange={selectedId => this.handleChange('isoImage', selectedId)}
+              validationState={indicators.isoList}
```

Comment:
If `indicators.isoList` will be a boolean value, too, we get a proptype error, too. Same with using `indicators.template` [here](https://github.com/oVirt/ovirt-web-ui/pull/1266/files#diff-cef40fc8d8b38f14df16a31106f8d660R382).